app/feature/feature-connect-payment-trustly/build.gradle.kts

```diff
@@ -12,26 +12,36 @@ hedvig {
 dependencies {
   api(libs.androidx.navigation.common)
 
-  implementation(libs.androidx.compose.material3)
+  implementation(libs.androidx.compose.animation)
```

Comment:
are you sure we need them all, androidx.compose? androidx.compose.foundation with the design system is usually enough